### PR TITLE
feat(normalize): add `jc normalize` flow

### PR DESCRIPTION
**Pull request by npitsillos**
**Goal**
- [x] Adds the `jc normalize` command to normalize a flow before deployment.

This PR also checks if a flow is normalized when `jc deploy` is called before a flow has been normalized.

- [x] Run [Integration tests GHA](https://github.com/jina-ai/jcloud/actions/workflows/integration-tests.yml) manually & comment the link. https://github.com/jina-ai/jcloud/actions/runs/3548789055

@jina-ai/team-wolf

### CODE REVIEW

**Comment by npitsillos** [comment]
In this PR I am also simplifying the `_get_post_params()` logic. See screenshots below.
### Before
image

### After
image (1)

File: jcloud/normalize_helper.py [hunk 1]
```diff
@@ -0,0 +1,308 @@
+import os
```
Comment (deepankarm) [inline jcloud/normalize_helper.py:1]: Please rename this file to `normalize.py`

**Review by deepankarm — CHANGES_REQUESTED** [review]
Minor comment

**Comment by npitsillos** [comment]
### Running `jc normalize <path>`
image

### Running `jc deploy <path>`
image

**Review by deepankarm — APPROVED** [review]
Great work :+1:

**Pull request merged**